runners/direct-java/build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
   shadow library.java.slf4j_api
```

Comment:
I still think we want to use `provided` for `beam-sdks-java-harness` so that it is correctly added to the generated maven pom.